cli/verifier/verifiers_test.go

```diff
@@ -0,0 +1,193 @@
+// Copyright 2025 Edgeless Systems GmbH
+// SPDX-License-Identifier: BUSL-1.1
+
+package verifier_test
+
+import (
+	"testing"
+
+	"github.com/edgelesssys/contrast/cli/verifier"
+	"github.com/edgelesssys/contrast/internal/kuberesource"
+	"github.com/stretchr/testify/require"
+)
+
+func TestVerifyNoSharedFSMount(t *testing.T) {
+	testCases := map[string]struct {
+		k8sObjectYAML string
+		wantErr       bool
+	}{
+		"unproblematic yaml": {
+			k8sObjectYAML: `
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test
```

Comment:
I find this very hard to read. Could we maybe move the yaml strings to constants outside the test body?